### ISSUE TITLE
ngx-datatable isnt working with scroll and limit

### ISSUE THREAD

**Issue by 42alex42** [42alex42/angular#1]
https://stackblitz.com/edit/ngx-datatable-scroll-with-limit

if start in stackblitz you see 5 rows with header and footer. I want to change the select-box above with the rows. if you select 15 it changes not now the table. But when you resize the browser, then you get the right 15 rows with header and footer. 
WHY is this only after resize the browser?
